package.json

```diff
@@ -41,6 +41,7 @@
     "react-router-dom": "^5.1.2",
     "react-select": "^1.0.0-rc.5",
     "react-tippy": "^1.2.2",
+    "swr": "^2.2.5",
```

Comment:
I'm curious why `swr` over `react-query` or something else comparable. Seems like a more compact toolkit than `react-query`, so I can understand if we want to explore needs with it before bringing in a heavier hitter. I've scanned most of the `swr` docs and it looks pretty cool, so I'm into it.

Reply:
good question, yea it is more lightweight and since we're only fetching data, and not doing any CUD, react-query seems more than we need for this project, but I'm not against standardizing around react-query in general